src/core/utils/otns.cpp

```diff
@@ -133,4 +134,10 @@ void Otns::EmitNeighborChange(otNeighborTableEvent aEvent, Neighbor &aNeighbor)
 } // namespace Utils
 } // namespace ot
 
+OT_TOOL_WEAK void otPlatOtnsStatus(const char *aStatus)
+{
+    OT_UNUSED_VARIABLE(aStatus);
+    otLogInfoPlat("%s\r\n", aStatus);
```

Comment:
```suggestion
    _otLogFormatter(OT_LOG_LEVEL_NONE, OT_LOG_REGION_CORE, "[OTNS] %s", aStatus, NULL);
```
- use log level `NONE` to make sure the log is printed despite the logging level? 
- Don't append `\r\n` to the end of format string. (logs are not supposed to contain the EOLs)
- Use some specific tags to make it easier for log parsers to identify OTNS status strings

Reply:
This will output the logs to `wpantund` at `[EMRG]` level. Would this be okay?